website/website/website.py

```diff
@@ -0,0 +1,80 @@
+import os
+from aiohttp import web
+import jinja2
+import logging
+import aiohttp_session
+
+from hailtop.config import get_deploy_config
+from hailtop.tls import internal_server_ssl_context
+from hailtop.hail_logging import AccessLogger
+from gear import setup_aiohttp_session, web_maybe_authenticated_user
+from web_common import (setup_aiohttp_jinja2, setup_common_static_routes, render_template,
+                        sass_compile)
+
+
+MODULE_DIR = os.path.dirname(__file__)
+log = logging.getLogger('scorecard')
+deploy_config = get_deploy_config()
+routes = web.RouteTableDef()
+
+
+@routes.get('/healthcheck')
+async def get_healthcheck(request):  # pylint: disable=unused-argument
+    return web.Response()
+
+
+def make_template_handler(template_fname):
+    @web_maybe_authenticated_user
+    async def serve(request, userdata):
+        return await render_template(
+            'website', request, userdata, template_fname, dict())
+    return serve
+
+
+routes.static('/docs/0.2/_static', f'{MODULE_DIR}/docs/0.2/_static')
+routes.static('/docs/batch/_static', f'{MODULE_DIR}/docs/batch/_static')
+
+
+@routes.get('/docs/{tail:.*\.html}')
+@web_maybe_authenticated_user
+async def serve_docs(request, userdata):
+    return await render_template('website', request, userdata, request.match_info['tail'], dict())
```

Comment:
This has a bad smell, I'm uncomfortable feeding arbitrary input (the URL) into jinja2.  Is it doing validation?  Can we be sure we're only pulling files from files from docs/ here. Letting a request pull an arbitrary template is probably not OK.